packages/react/src/cunningham-tokens.scss

```diff
@@ -143,6 +143,21 @@ $themes: (
         'icon-size': 19px,
         'progress-bar-height': 3px
       ),
+      'modal': (
+        'background-color': #FFFFFF,
+        'border-radius': 4px,
+        'border-color': #E7E8EA,
+        'box-shadow': 0px 1px 2px 0px #0C1A2B4D,
+        'color': #0C1A2B,
+        'backdrop-color': #0C1A2B99,
+        'title-font-weight': 600,
+        'content-font-size': 0.8125rem,
+        'content-font-weight': 400,
+        'content-color': #303C4B,
+        'small-width': 300px,
+        'medium-width': 600px,
+        'large-width': 75%
```

Comment:
To improve DX, it's better to start the token name by the property then the variant.
In this way, by typing `wid`, autocompletion will show all tokens related to `width` property.